src/policy/mod.rs

```diff
@@ -35,22 +35,27 @@ pub use self::concrete::Policy as Concrete;
 /// Semantic policies are "abstract" policies elsewhere; but we
 /// avoid this word because it is a reserved keyword in Rust
 pub use self::semantic::Policy as Semantic;
+use MiniscriptKey;
 
 /// Trait describing script representations which can be lifted into
-/// an abstract policy, by discarding information
-pub trait Liftable<Pk, Pkh> {
+/// an abstract policy, by discarding information.
+/// After Lifting all policies are converted into `KeyHash(Pk::HasH)` to
+/// maintain the following invariant:
+/// `Lift(Concrete) == Concrete -> Miniscript -> Script -> Miniscript -> Semantic`
+pub trait Liftable<Pk: MiniscriptKey> {
+>>>>>>> 027a708... Refactor miniscript Key
```

Comment:
Can you remove this `>>>>>`? I guess it's removed in a later commit?

Please use `git rebase -x 'cargo test'` to check that all of your commits compile

Reply:
Nice, I was manually checking out all the commits till now. that's pretty handy